components/nimbus/ios/Nimbus/Bundle+.swift

```diff
@@ -68,12 +68,13 @@ public extension Bundle {
     /// If `language` is `nil`, then look for the development region language.
     /// If no bundle for the language exists, then return `nil`.
     func fallbackTranslationBundle(language: String? = nil) -> Bundle? {
-        if let lang = language ?? infoDictionary?["CFBundleDevelopmentRegion"] as? String,
-           let path = path(forResource: lang, ofType: "lproj")
-        {
-            return Bundle(path: path)
-        } else {
-            return nil
-        }
+        #if canImport(UIKit)
```

Comment:
Searching has indicated this is a compiler test which is pretty cool!

Our library publishes the actual Swift files, so the libraries that use it would then be responsible for compiling it?

Reply:
I believe the swift files aren't actually compiled until the Swift Package Manager module is built with the app. The rust bits are already built and shipped along with the Swift package as a compiled library binary

Reply:
This is why we have to copy all the Swift files when building the iOS megazords in A-S

Reply:
Gotcha! Neat I just hadn't seen swift's compiler testing before